src/Objs/RedirectToUi/RedirectToUiComponent.js

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+
+class RedirectToUiComponent extends React.Component {
+  componentDidMount() {
+    const scrivitoUiUrl = window.location.origin + "/scrivito/";
```

Comment:
* [ ] FYI `origin` is IE11+ only https://developer.mozilla.org/en-US/docs/Web/API/Window/location

Reply:
Our UI does not even support IE11. To quote from https://www.scrivito.com/user-guide:

> Supported browsers for editing
> * Google Chrome: latest 2 versions
> * Mozilla Firefox: latest 2 versions
> * Apple Safari: latest version
> * Microsoft Edge: latest version

Reply:
I see, didn't have in mind that this is a UI feature of the app. 😝